File: platform/sdk/nodejs/examples/cloud/cloud-005_set_bucket_acl.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("acl-bucket");
const result = client.setBucketAcl("acl-bucket", "public-read");
if (!result.ok || result.acl !== "public-read") process.exit(1);
console.log("XJ-OK: cloud-005_set_bucket_acl");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-005_set_bucket_acl.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("acl-bucket")
result = client.setBucketAcl("acl-bucket", "public-read")
if not result["ok"] or result["acl"] != "public-read":
    sys.exit(1)
print("XJ-OK: cloud-005_set_bucket_acl")
sys.exit(0)
